.github/workflows/build-artifacts.yml

```
@@ -0,0 +1,74 @@
+name: build-artifacts.yml
+on:
+  workflow_dispatch:
+  push:
+    branches: [ main, SERVER-216,  SERVER-434 ]
```

Do we want to run the build only on merge to main? Wouldn't we need to do something on PR also (either just build or build and test)

in either case should remove the branches
```suggestion
    branches: [ main ]
```

---

I changed it build on this branch, and on tag creation. 

Lets talk about this with QE, anyway removing the SERVER-216 should be the last commit here before we merge